components/NavBar.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <nav>
+    <div class="title">
+      <h1>GitHub Issue Viewer</h1>
+    </div>
+    <div class="url">
+      <h2>
+        {{
+          `https://github.com/${this.$route.params.owner}/${this.$route.params.repo}`
+        }}
+      </h2>
+    </div>
+  </nav>
+</template>
+
+<script>
+export default {
+  computed: {
+    gitUrl: function() {
```

Comment:
This works, but you can write this a bit more elegant:

`computed: mapState(['searchedUrl'])`

Then, you can use `this.searchedUrl` as if it was a computed variable. 

https://vuex.vuejs.org/guide/state.html